services/tiktok.yml

```
@@ -25,4 +25,8 @@ rules:
   - '||tiktok.com^'
   - '||tiktokcdn.com^'
   - '||tiktokv.com^'
+  - '||tiktokcdn-us.com^'
+  - '||ttlivecdn.com.c.bytefcdn-oversea.com^'
+  - '||ttlivecdn.com^'
+  - '||p16-tiktok-*.ibyteimg.com^'
```

Maybe they have another sub-domains
 ```suggestion
  - '-tiktok-*.ibyteimg.com^'
```

---

> Not sure about

Yes, they are too wide and not only Tiktok will be blocked.